machines/settings.ts

```diff
@@ -352,3 +380,15 @@ export function selectIsResetInjiProps(state: State) {
 export function selectIsBackUpAndRestoreExplored(state: State) {
   return state.context.isBackupAndRestoreExplored;
 }
+
+export function selectIsBiometricUnlock(state: State) {
+  return (
+    state.context.isBiometricToggled && state.context.isBiometricUnlockEnabled
+  );
+}
+
+export function selectIsPasscodeUnlock(state: State) {
+  return (
+    state.context.isBiometricToggled && !state.context.isBiometricUnlockEnabled
```

Comment:
here state.context.isBiometricToggled  this should be negation right? Biometric Toggle should be false?

Reply:
here isBiometricToggled value is set to true everytime there is change in value from the settings biometric switch. Deafult it will be false.